health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/BaseMapSwitcher.js

```
@@ -0,0 +1,59 @@
+
+const BaseMapSwitcher = ({ baseMaps, showBaseMapSelector, setShowBaseMapSelector, handleBaseMapToggle, selectedBaseMapName, basemapRef, t }) => {
+  if (!baseMaps) return null;
+  return (
+    <div className="base-map-selector">
+      <div
+        className="icon-first"
+        onClick={() => setShowBaseMapSelector((previous) => !previous)}
+        onKeyUp={() => setShowBaseMapSelector((previous) => !previous)}
+        tabIndex={0}
+        style={{display:"flex"}}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)